snapcraft/extensions/ros2_humble.py

```diff
@@ -120,8 +121,13 @@ def get_parts_snippet(self) -> Dict[str, Any]:
             f"ros2-{self.ROS_DISTRO}/ros2-launch": {
                 "source": f"{get_extensions_data_dir()}/ros2",
                 "plugin": "nil",
-                # pylint: disable=line-too-long
-                "override-build": "install -D -m 0755 launch ${CRAFT_PART_INSTALL}/snap/command-chain/ros2-launch",
+                #  disable=line-too-long
+                "override-build": dedent(
+                    """\
+                    install -D -m 0755 launch ${CRAFT_PART_INSTALL}/snap/command-chain/ros2-launch
+                    install -D -m 0755 ros2 ${SNAPCRAFT_PART_INSTALL}/ros2
```

Comment:
Shouldn't this also use `CRAFT_PART_INSTALL` ?

Reply:
`SNAPCRAFT_PART_INSTALL` is still around but you're right, I'll fix that :+1:

Reply:
'Fixed' in 91e1a1eacbceb10e1539d19d4d7d53fd1c9585ae.